src/catalogue/category/catalogueCard.component.tsx

```diff
@@ -83,21 +96,64 @@ function CatalogueCard(props: CatalogueCardProps) {
           <IconButton
             onClick={(event) => {
               event.preventDefault();
-              onChangeOpenEditDialog(catalogueCategory);
+              setAnchorEl(event.currentTarget);
+              setMenuOpen(true);
             }}
-            aria-label={`edit ${catalogueCategory.name} catalogue category button`}
+            aria-label={`actions ${catalogueCategory.name} catalogue category button`}
           >
-            <EditIcon />
+            <MoreHorizIcon />
           </IconButton>
-          <IconButton
-            onClick={(event) => {
-              event.preventDefault();
-              onChangeOpenDeleteDialog(catalogueCategory);
-            }}
-            aria-label={`delete ${catalogueCategory.name} catalogue category button`}
+          <Menu
+            anchorEl={anchorEl}
+            open={menuOpen}
+            onClose={handleActionsClose}
           >
-            <DeleteIcon />
-          </IconButton>
+            <MenuItem
+              key={0}
+              onClick={(event) => {
+                event.preventDefault();
+                onChangeOpenEditDialog(catalogueCategory);
+                setMenuOpen(false);
```

Comment:
```suggestion
                handleActionsClose();
```
I think if you do this it will resolve the 1 line missing coverage.